src/test/java/org/checkerframework/specimin/LambdaBiFunctionTest.java

```diff
@@ -0,0 +1,18 @@
+package org.checkerframework.specimin;
+
+import java.io.IOException;
+import org.junit.Test;
+
+/**
```

Comment:
Is it intentional that all the tests here have the same Javadoc?

Reply:
I was being lazy :) thanks for calling me out on this, I'll differentiate them so that it's more clear what each test is supposed to cover.